src/smk/tool/geomark/tool-geomark.js

```diff
@@ -102,6 +120,13 @@ include.module( 'tool-geomark', [
                 }
             } )
 
+            this.updateAndShowAlert = function(alertBodyArg) {
+                self.alertBody = alertBodyArg;
+                self.showAlert = true;
+            }
+
+            this.handleAlert = function() {};
```

Comment:
JavaScript has no concept of typings, so you don't need to assign an empty function to this property. This results in some additional overhead (though very small) that isn't necessary. You can simply set `this.handleAlert = undefined` here and on line 233.

Reply:
And maybe add a comment stating the purpose of the handleAlert property.

Reply:
Done.